devito/operator.py

```diff
@@ -304,7 +316,8 @@ def _schedule_expressions(self, clusters):
                 needed = entries[index:]
 
                 # Build and insert the required Iterations
-                iters = [Iteration([], j.dim, j.dim.symbolic_size, offsets=j.ofs)
+                iters = [Iteration([], j.dim, j.dim.limits,
+                                   offsets=j.ofs if j.dim.is_Fixed else (0, 0))
```

Comment:
why?

Reply:
If you're hardcoding the limits, hard-code the offsets as well. Else, the offset will be accounted for in the runtime limit calculation.

Reply:
no but I mean, why `j.ofs` wouldn't return ``(0, 0)`` if ``j.is_Fixed`` is True?

Reply:
Because it would return the offset in that dimension which depends on the stencil radius?

Reply:
anyway, we can drop this because of #373

Reply:
This will go in first

Reply:
@FabioLuporini - do you mean "drop the patch" or "drop the review comment"?

Reply:
I was assuming #373 would go in first (it's ready, while this one is not); if #373 goes in first, we can drop the ``is_Fixed`` check above, because #373 removes ``FixedDimension``